visivo/parsers/mkdocs_utils/markdown.py

```diff
@@ -0,0 +1,154 @@
+        enum_model_key = find_refs(attribute_property_object)[0].split("/")[-1]
+        enum_model = model_defs.get(enum_model_key, {})
+        if not enum_model:
+            raise KeyError(f"Key {enum_model_key} was not found in $defs dictionary.")
+        type = "Enumerated - one of: " + ", ".join(enum_model.get("enum"))
+    elif "anyOf" in attribute_key_type:
+        default = attribute_property_object.get("default", "None")
+        anyOf = find_refs(attribute_property_object)
+        for option in attribute_property_object["anyOf"]:
+            if "pattern" in option and find_refs(attribute_property_object) == []:
+                anyOf.append(f"Regex({option.get('pattern')})")
+            elif "type" in option and "pattern" not in option:
+                if option.get("type") not in ["null", "None"]:
+                    anyOf.append(option.get("type"))
+            else:
+                continue
+        if len(anyOf) > 1:
+            type = "Any of: " + ", ".join(anyOf)
+        elif len(anyOf) == 1:
+            type = anyOf[0]
+    elif "const" in attribute_key_type:
+        default = attribute_property_object.get("default", "None")
+        type = "string"
+    elif "items" in attribute_key_type:
+        default = attribute_property_object.get("default", "None")
+        type = attribute_property_object.get("type")
+        refs = find_refs(attribute_property_object.get("items", {}))
+        if type == "array" and refs:
+            type = "Array of " + " or ".join(refs)
+    elif "discriminator" in attribute_key_type:
+        default_key = attribute_property_object.get("discriminator", {}).get(
+            "propertyName", ""
+        )
+        default_value = attribute_property_object.get("default", {}).get(default_key)
+        default = (
+            attribute_property_object.get("discriminator", {})
+            .get("mapping", {})
+            .get(default_value)
+        )
+        refs = find_refs(attribute_property_object.get("oneOf", []))
+        if refs:
+            type = "One of:<br>  •" + "<br>  •".join(refs)
+    else:
+        default = attribute_property_object.get("default", "None")
+        type = attribute_property_object.get("type", "")
+    default = default if default is not None else "None"
+
+    return type, description, default
+
+
+def from_pydantic_model(model_defs: dict, model_name: str) -> str:
+    """Generates markdown from the model description and creates table of attributes from $defs in pydantic schema"""
+    model_def = model_defs.get(model_name, {})
+    if not model_def:
+        raise KeyError(f"Schema missing model: {model_name}")
+    model_properties = model_def.get("properties", {})
+    model_md = (
+        ""
+        if not model_def.get("description", {})
+        else dedent(model_def.get("description"))
+    )
+    md_table = "| Field | Type | Default | Description |\n|-------|------|---------|-------------|\n"
```

Comment:
Can use the "join" function for spring building:

`" | ".join("Field", "Type", "Default", "Description")`